src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1207,32 +1208,119 @@ void missingRdfTypeOnNamedNode() {
         }
     }
 
+    class BaseExample {
+        Model model;
+        Resource system;
+        ValidateAIF validator = seedlingValidator;
+
+        void setup() {
+            model = createModel();
+            system = makeSystemWithURI(model, getTestSystemUri());
+        }
+
+        void addType(Resource resource, Resource type) {
+            markType(model, getAssertionUri(), resource, type, system, 1d);
+        }
+
+        void testInvalid(String name) {
+            assertAndDump(model, name, validator, false);
+        }
+
+        void testValid(String name) {
+            assertAndDump(model, name, validator, true);
+        }
+
+    }
+
     /**
      * Set of tests to show that NIST restrictions pass and fail appropriately
      */
     @Nested
-    class NISTExamples {
-        Model model;
-        Resource system;
+    class NISTExamples extends BaseExample {
```

Comment:
As a group @ecurley-nextcentury , @nextcen-dgemoets, @sharknc we should decide if we like this design and if we want to move forward with extending all nested test classes with BaseExample. Currently NISTExample is the only class that extends BaseExample. My vote is to keep it, but just update the rest of the test classes to make everything uniform.

Reply:
I agree, and moreover I think that ultimately we'll want NISTHypothesisExamples to subclass NISTExamples because soon they'll need events and relations.

Reply:
BaseExample was removed in [`d72c1`](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/pull/134/commits/d72c1df86a351b1aed6e80ad7a548e69ea454427) in favor of moving its functionality to the ExamplesAndValidationTest class.